README.md

```diff
@@ -74,6 +136,17 @@ REPL features:
     ![repl completion](http://i.imgur.com/IXLQFtV.gif)
 * Exit the REPL by writing `:q`
 
+### Tests
+
+Gluon has extensive tests. Look at `scripts/travis.sh` for extensive examples, but as a quick start 
+
+```bash
+cargo test --features test
+cargo test --features test --all
```

Comment:
This is actually covered in [CONTRIBUTING.md](https://github.com/gluon-lang/gluon/blob/master/CONTRIBUTING.md) (which I could have pointed to yesterday). Do you feel like it is good to have it up here as well? If a separate file seems "too far away" perhaps it could be located in the `Contributing` section instead as a short message before the link to the actual file?

Reply:
I missed that testing is covered in CONTRIBUTING.md, but part of the idea of adding this to README was also that instructions/help on running tests are visible at first glance (I actively look for it). I quite like that idea of keeping the information in Contributing but adding a note in readme, so I will update the PR in that direction.